runtime/ms_rest_azure/spec/polling_state_spec.rb

```diff
@@ -10,25 +10,31 @@ module MsRestAzure
   describe PollingState do
     it 'should initialize status from flattened response body' do
       response_body = double('response_body', :provisioning_state => 'InProgress')
-      response = double('response',
```

Comment:
was the test before these changes valid? By this question I mean, would it be valid to have a nil "response" with a response body?